packages/block-library/src/comment-template/index.php

```diff
@@ -53,25 +53,17 @@ function block_core_comment_template_render_comments( $comments, $block ) {
  * defined by the block's inner blocks.
  */
 function render_block_core_comment_template( $attributes, $content, $block ) {
-
-	$post_id = $block->context['postId'];
-
 	// Bail out early if the post ID is not set for some reason.
-	if ( ! isset( $post_id ) ) {
+	if ( empty( $block->context['postId'] ) ) {
```

Comment:
Why do we not have to check if `postId` is set? 🤔

Reply:
In this case, `empty()` is checking if postId exists. 😄  [More info here](https://www.php.net/manual/en/function.empty.php)

Reply:
From the linked document:

> No warning is generated if the variable does not exist. That means `empty()` is essentially the concise equivalent to `!isset($var) || $var == false`.

In general, in PHP all variables (including deeply nested array/object access) should always be guarded with `isset` /`empty` check. I saw in other PRs that we sometimes assume that some keys inside the `$block->context` are always present that might be not true.

Reply:
Thank you guys 🙏  TIL!